pkg/extended_registry/ghcr_test.go

```diff
@@ -0,0 +1,195 @@
+package extendedregistry
+
+import (
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/aserto-dev/go-grpc/aserto/api/v1"
+	"github.com/rs/zerolog"
+	"gopkg.in/h2non/gock.v1"
+	"gotest.tools/assert"
```

Comment:
we tipically use `github.com/stretchr/testify/require` instead of `gotest.tools/assert`. For example: https://github.com/opcr-io/policy/blob/main/pkg/app/build_test.go#L8
It would be great to be consistant.